server/index.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+console.log("Launching");
+const app = require( './app' );
+console.log( "App.js" );
+const models = require( './models/index' );
+console.log("Loading models");
+
+/* Start the server */
+const http = require( 'http' ).Server( app ); 
+
+
+
+/* Warn about environment variables */
+const warnEnvVars = function( vars ) {
+  let didWarn = false;
+  let warnString = '';
+  vars.forEach( function( envVar ) {
+    if( !process.env[envVar] ) {
+      didWarn = true;
+      warnString += '\n* It may be a good idea to set the ' + envVar + ' environment variable. You can use `export ' + envVar + '=<text>` to do this.';
+    }
+  });
+
+  if( didWarn ) {
+    console.log( 'WARNING:', warnString );
+  }
+}
+
+warnEnvVars( ['PORT', 'JWT_SECRET', 'SQLUSER', 'SQLPASS', 'SQLPORT', 'SQLDBNAME'] );
+
+process.env.JWT_SECRET  = process.env.JWT_SECRET || 'asf23r9sdf21';
+
+/* Set the port, default 3000 */ 
+app.set( 'port', process.env.PORT || 3000 );
+
+const initTableData = require( './lib/initTableData' );
+console.log("Initializing table data");
```

Comment:
This console.log should go near the line when the initTableData function is actually executed-- after line 45.